binder-index.md

```diff
@@ -99,6 +99,14 @@ These are noted in the README.md files for each sample, along with complete inst
       <td>N/A</td>
       <td></td>
     </tr>
+    <tr>
+      <td></td>
+      <td><strong><a href="./samples/getting-started/simulation/README.md">Simulation</a></strong></td>
+      <td><a href="./samples/getting-started/simulation/LargeSimulation.ipynb">Q# notebook</a></td>
```

Comment:
This should be fine for now, but may need some rearranging once we get other basic simulator samples.